.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/stale
+
+name: Close Inactive Issues and Pull Requests
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 90
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          exempt-issue-labels: ("release Q3-2023", "release Q4-2023", "release Q1-2024")
```

Comment:
Do we need this? Our release cycle length is 90 days. If an issue with release label gets stale, it indicates most likely a problem.